nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -198,6 +201,25 @@ public void onTrigger(final ProcessContext context, final ProcessSessionFactory
             // set as the current state map (after the session has been committed)
             final Map<String, String> statePropertyMap = new HashMap<>(stateMap.toMap());
 
+            // If an initial max value for column(s) has been specified using properties, and this column is not in the state manager, sync them to the state property map
+            final Map<String,String> maxValueProperties = getDefaultMaxValueProperties(context.getProperties());
```

Comment:
Would it make sense to do it in the ``@OnScheduled`` method of the Abstract class? You could have the same code in common for QueryDatabaseTable and GenerateTableFetch. Besides not sure it needs to be done each time the processor is triggered? Thoughts?